src/main/webapp/web.xml

```diff
@@ -0,0 +1,14 @@
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+         http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+         version="3.1">
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>taxi.controllers.IndexController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```

Comment:
Same.
Don't know if it's necessary here.

Reply:
Just for the sake of interest)
<img width="658" alt="Screenshot 2021-01-19 at 20 48 17" src="https://user-images.githubusercontent.com/67925366/105079128-aa615880-5a97-11eb-8ded-545a2b68e797.png">